src/App.scss

```diff
@@ -0,0 +1,24 @@
+.app {
```

Comment:
- [ ] Why is it using both scss and css files? I think its only using css files here. So use just one of them and delete the rest

Reply:
Seems like its using the scss to css mapping here which is giving us more files in terms of rendering. Need to make sure how its better than using css modules.

Reply:
Forgot to commit the common css rules and normalizations to master.  So had to add these in every branch